### DISCUSSION TITLE
Upcoming FINOS Meetup - How To Build An Order Book Simulation Using Perspective

### DISCUSSION THREAD

**Discussion by texodus** [perspective-dev/perspective#1366] [General]
I'll be giving a short talk on Perspective via zoom teleconference next week for FINOS, based on a new example application we're working on for the upcoming `0.7.0` release - and interactive streaming [Order Book Simulation](https://github.com/finos/perspective/pull/1277).

FINOS OPEN SOURCE IN FINTECH MEETUP - HOW TO BUILD AN ORDER BOOK SIMULATION WITH PERSPECTIVE
March 31, 2021, 11am EDT / 4pm BST
Online - Zoom
[Register here!](https://www.finos.org/hosted-events/2021-03-31-os-in-fintech-meetup-perspective)

<img width="1842" alt="Screen Shot 2021-03-04 at 3 57 11 PM" src="https://user-images.githubusercontent.com/60666/112706817-2d7aa280-8e7d-11eb-892d-5411fec0cbc3.png">

**Comment by mcleo-d** [comment]
Fantastic presentation and demo @texodus!

Thanks for taking the time to guide the FINOS community through Perspective. 

It's really awesome to see https://github.com/finos/perspective/issues/1373 raised by @f0ster as a result 🚀 

Congratulations 👏 💯 💯 💯 

James.

**Comment by mcleo-d** [comment]
@texodus - Just to note, there is a typo in your discussion title ...

How To Build An Order Book Simulation Using `Perspectinve`

... _should be_ 

How To Build An Order Book Simulation Using `Perspective`

😉
